### PR TITLE
fix: Make socialLinksRecycler work properly.

### LINKED ISSUE #1090: Event host details (i.e socialLinks) not showing.
**Describe the bug**
Social Links in event host details not showing up.

**To Reproduce**
1. Go to any event which has social links
2. Links won't be visible under Event host details under Organizer.

**Expected behavior**
Social Links should be visible.

**Logs**
Shows that event organizer had social links:

2019-02-13 18:19:21.116 20334-20334/com.eventyay.attendee E/yo: Twitterhttp://twitter.com/fossasia
2019-02-13 18:19:21.116 20334-20334/com.eventyay.attendee E/yo: Facebookhttp://facebook.com/fossasia
2019-02-13 18:19:21.116 20334-20334/com.eventyay.attendee E/yo: Bloghttp://blog.fossasia.org
2019-02-13 18:19:21.116 20334-20334/com.eventyay.attendee E/yo: YouTubehttps://www.youtube.com/channel/UCQprMsG-raCIMlBudm20iLQ
2019-02-13 18:19:21.116 20334-20334/com.eventyay.attendee E/yo: Flickrhttps://www.flickr.com/photos/fossasia/
2019-02-13 18:19:21.116 20334-20334/com.eventyay.attendee E/yo: GitHubhttps://github.com/fossasia
2019-02-13 18:19:21.116 20334-20334/com.eventyay.attendee E/yo: LinkedInhttps://www.linkedin.com/groups/FOSSASIA-3762811
2019-02-13 18:19:21.116 20334-20334/com.eventyay.attendee E/yo: Googlehttp://plus.google.com/108920596016838318216
2019-02-13 18:19:21.116 20334-20334/com.eventyay.attendee E/yo: Wikipediahttp://en.wikipedia.org/wiki/FOSSASIA

**Screenshots**
image

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by angmas1**
Fixes #1090 

Changes: Improved search for link name by adding more options. Added drawable resource for some more popular sites. Added a drawable  for when link is of some other site.

Screenshots for the change:
image

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Use imperative present tense in the PR title. See the examples in merged PRs. Apply the same rule to all your PRs

**Comment by angmas1** [comment]
> Use imperative present tense in the PR title. See the examples in merged PRs. Apply the same rule to all your PRs

Done

File: app/src/main/java/org/fossasia/openevent/general/social/SocialLinksViewHolder.kt [hunk 1]
```diff
@@ -15,28 +15,55 @@ class SocialLinksViewHolder(itemView: View, private var context: Context) : Recy
...
+            "flickr",
+            "flickr url",
+            "flickrurl" -> R.drawable.ic_flickr
+            "blogger",
+            "blogger url",
+            "bloggerurl",
+            "blog",
+            "blog url",
+            "blogurl" -> R.drawable.ic_blogger
+            else -> R.drawable.ic_link_black
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/social/SocialLinksViewHolder.kt]: Use `if (name.toLowerCase().includes("github"))` and so on

Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/social/SocialLinksViewHolder.kt]: Thanks for reminding, I forgot about includes function.

Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/social/SocialLinksViewHolder.kt]: done

File: app/src/main/java/org/fossasia/openevent/general/social/SocialLinksViewHolder.kt [hunk 2]
```diff
@@ -26,44 +26,15 @@ class SocialLinksViewHolder(itemView: View, private var context: Context) : Recy
...
-        }
+        if (name.toLowerCase().contains("github")) return R.drawable.ic_github
+        else if (name.toLowerCase().contains("twitter")) return R.drawable.ic_twitter
+        else if (name.toLowerCase().contains("facebook")) return R.drawable.ic_facebook
+        else if (name.toLowerCase().contains("linkedin")) return R.drawable.ic_linkedin
+        else if (name.toLowerCase().contains("youtube")) return R.drawable.ic_youtube
+        else if (name.toLowerCase().contains("google")) return R.drawable.ic_google_plus
+        else if (name.toLowerCase().contains("wiki")) return R.drawable.ic_wikipedia
+        else if (name.toLowerCase().contains("flickr")) return R.drawable.ic_flickr
+        else if (name.toLowerCase().contains("blog")) return R.drawable.ic_blogger
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/social/SocialLinksViewHolder.kt]: Extract name.toLowerCase to another variable

Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/social/SocialLinksViewHolder.kt]: okay

Comment (angmas1) [reply app/src/main/java/org/fossasia/openevent/general/social/SocialLinksViewHolder.kt]: done

**Pull request merged**